examples/psp22/lib.rs

```diff
@@ -9,20 +9,34 @@ pub mod my_psp22 {
         contracts::psp22::*,
         traits::{
             Storage,
+            StorageAsMut,
             String,
         },
     };
 
-
     #[ink(storage)]
     #[derive(Storage)]
     pub struct Contract {
         #[storage_field]
         psp22: psp22::Data,
         // fields for hater logic
+        #[storage_field]
+        hated_logic: HatedLogic,
+    }
+
+    #[openbrush::upgradeable_storage(STORAGE_KEY)]
+    #[openbrush::accessors(HatedLogicAccessors)]
+    #[derive(Debug)]
+    pub struct HatedLogic {
+        #[get]
+        #[set]
```

Comment:
can we add two more fields, one with only get and one with only set? and test them too

Reply:
added more tests in #66